lib/utils/webpack.config.js

```diff
@@ -117,6 +117,7 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
             },
             __PREFIX_LINKS__: program.prefixLinks,
           }),
+          new ExtractTextPlugin('styles.css'),
```

Comment:
We don't extract styles here and in `build-javascript` on purpose as we already do that in the `build-css` stage. Doing it again would be wasted effort.